internal/ingest/linksystem.go

```diff
@@ -348,6 +355,8 @@ func (ing *Ingester) ingestAd(publisherID peer.ID, adCid cid.Cid, ad schema.Adve
 				return adIngestError{adIngestSyncEntriesErr, fmt.Errorf("failed to sync entries: %w", err)}
 			}
 		}
+
+		log = log.With("ingestTotalTime", ingestTotalTime/1000.0).With("totalChunksCount", totalChunksCount)
```

Comment:
This is a great start thanks @ischasny 👍 
* I think we need grafana metrics, so it would be great to report them that way too.
* we need metrics for both HAMT and EntryChunk processing time, tagged by entry kind.

Reply:
Yes, agreed. That was a quick attempt to understand where the problem is, not the final solution by any means :)